components/LogWorkout/LogWorkout.jsx

```diff
@@ -76,6 +86,49 @@ const LogWorkout = (props) => {
 
   const [exerciseState, setExerciseState] = useState(initialExerciseState);
 
+  const currentUser = firebase.auth().currentUser.uid;
+  // const currentUser = '68w6wWz8l5QJO3tDukh1fRXWYjD2';
+
+  const dbRef = firebase.firestore();
+  const userRef = dbRef.collection('users').doc(currentUser);
+  const dispatch = useDispatch();
+
+  const incrementSelectedCycleIdx = () => {
+    dispatch({ type: INCREMENT_SELECTED_CYCLE_INDEX, cycleLength });
+    userRef.update({ selectedCycleIndex: (cycleIdx + 1) % cycleLength });
+  };
+
+  const sendWorkoutLogToDB = () => {
+    const workoutRecsRef = userRef.collection('workoutRecords');
+    const { format } = require('date-fns');
```

Comment:
Can you change this to an import at the top of the code

Reply:
sorry dont understand what u want

Reply:
instead of `const { format } = require('date-fns')` here in the middle of your code move it to the top and import it like all of the other modules used in this file. `import { format } from 'date-fns'`